src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingService.cs

```diff
@@ -206,34 +225,26 @@ public bool TryGetOnTypeFormattingTriggerKind(RazorCodeDocument codeDocument, in
     }
 
     private async Task<ImmutableArray<TextChange>> ApplyFormattedChangesAsync(
-        DocumentContext documentContext,
+        IDocumentSnapshot documentSnapshot,
+        RazorCodeDocument codeDocument,
         ImmutableArray<TextChange> generatedDocumentChanges,
         RazorFormattingOptions options,
         int hostDocumentIndex,
         char triggerCharacter,
         ImmutableArray<IFormattingPass> formattingPasses,
         bool collapseChanges,
-        bool isCodeActionFormattingRequest,
+        bool automaticallyAddUsings,
```

Comment:
@phil-allen-msft you were absolutely right to complain about the naming of this variable, and there is a slim chance that if you'd pushed harder, I might have thought more about it and realised the association it was making was insufficient, and there was more to fix than just code actions.

ie, it's all your fault 😛